Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemController.java

```diff
@@ -94,6 +96,13 @@ public ProblemView getProblem(@PathVariable int problemId) {
                 .execute(new DownloadTestCaseIOsUseCase.Request(problemId, testcaseIOsFileId));
         return ResponseEntityUtils.respondInputStreamResource(fileResource);
     }
+
+    @PostMapping(value = "/title", consumes = "text/plain")
+    @ResponseStatus(value = HttpStatus.OK)
```

Comment:
We don't need this line.
It will be 200 if this method is completed successfully